### PR TITLE
FOX-173-Implement-Anchor-terrain-divots-and-particle-effects

### PR DESCRIPTION
**Pull request by sachdawg**
test scene: Brian level idea or Harley level 1

## Changes summary:
- made anchor impact crater and rock debris particles

## Checklist before requesting a review:
- [x] I have run the game locally
- [x] I have performed self-review on my code
- [x] I have confirmed critical features still function

### CODE REVIEW

File: Assets/Scripts/FadeAndDestroy.cs [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+    private void Awake()
+    {
+        m_SpriteRenderers = GetComponentsInChildren<SpriteRenderer>();
+
+        m_FadeOutDelay = 0; //Random.Range(0.5f, 1.5f);
+        m_FadeOutLength = 0.2f; //Random.Range(0.2f, 0.5f);
+
+        m_OriginalScale = transform.localScale;
+        Debug.Log(m_OriginalScale);
```
Comment (jess-rubino) [inline Assets/Scripts/FadeAndDestroy.cs:31]: might want to remove this debug

Comment (sachdawg) [reply Assets/Scripts/FadeAndDestroy.cs:31]: oopsyy

**Review by jess-rubino — APPROVED** [review]
LGTM

**Comment by ShiroKatake** [comment]
Link to build: https://github.com/WeAreBrian/project-fox-poc/actions/runs/5916998855

**Pull request merged**